stack.yaml

```diff
@@ -42,18 +42,12 @@ extra-deps:
 - servant-swagger-1.1.6
 - swagger2-2.3
 - git: https://github.com/wireapp/prometheus-haskell
```

Comment:
Since `wireapp` and `fimap` are now in sync, how about we pin to `fimap` instead?